core-bundle/src/Resources/contao/modules/Module.php

```diff
@@ -313,8 +312,10 @@ protected function renderNavigation($pid, $level=1, $host=null, $language=null)
 				$objSubpage->domain = $host;
 			}
 
+			$groups = StringUtil::deserialize($objSubpage->groups, true);
+
 			// Do not show protected pages unless a front end user is logged in
-			if (!$objSubpage->protected || $this->showProtected || ($this instanceof ModuleSitemap && $objSubpage->sitemap == 'map_always') || ($user && $user->isMemberOf(StringUtil::deserialize($objSubpage->groups))))
+			if (!$objSubpage->protected || $this->showProtected || ($this instanceof ModuleSitemap && $objSubpage->sitemap == 'map_always') || (!$user && \in_array(-1, $groups)) || ($user && $user->isMemberOf($groups)))
```

Comment:
Same here, I don’t understand how this could have worked before without calling `->loadDetails()` 🤔

Reply:
Me neither. Could this be a bug that already exists in Contao 4.9?

Reply:
I have added the `loadDetails()` call in 511b137ff74ec28f86d9e59ee13cd348db5fd493. We should still check Contao 4.9 and 4.11.